crates/api/src/content.rs

```diff
@@ -0,0 +1,22 @@
+//! Types relating to the content API.
+
+use serde::{Deserialize, Serialize};
+use warg_crypto::hash::DynHash;
+
+/// Represents a content source.
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
```

Comment:
Why the switch to camel case?

Reply:
Given it's a JSON-serialized format, I figured it'd be more natural to default to with these types (most places are a singular word, so it doesn't impact them, but setting it everywhere guards against the future).

Happy to revert if that doesn't make sense to do.

Reply:
I don't have a strong opinion here, and just picked a default option I tend to like.

Looking at the OCI distribution spec, they're using camel case.
Being consistent with them (and presumably many others) seems like a good enough reason to me.